Source/JavaScriptCore/wasm/WasmSlowPaths.cpp

```diff
@@ -1087,6 +1088,28 @@ WASM_SLOW_PATH_DECL(rethrow)
     WASM_RETURN_TWO(vm.targetMachinePCForThrow, nullptr);
 }
 
+WASM_SLOW_PATH_DECL(throw_ref)
+{
+    SlowPathFrameTracer tracer(instance->vm(), callFrame);
+
+    JSGlobalObject* globalObject = instance->globalObject();
+    VM& vm = globalObject->vm();
+    auto throwScope = DECLARE_THROW_SCOPE(vm);
+
+    auto instruction = pc->as<WasmThrowRef>();
+    JSWebAssemblyException* exception = reinterpret_cast<JSWebAssemblyException*>(READ(instruction.m_exception).pointer());
+
+    if (!exception)
+        WASM_THROW(Wasm::ExceptionType::NullExnReference);
```

Comment:
It it actually null pointer? Or `jsNull()`? (If it is `jsNull()`, we should `READ(instruction.m_exception)` as EncodedJSValue.
Also, I saw this exception only in LLInt's throw_ref. Don't we need to do this kind of handling in BBQ and OMG?